Directory.Build.props

```diff
@@ -3,31 +3,11 @@
 <Project>
 
   <PropertyGroup>
-    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileDirectory)..\Directory.Build.props</MSBuildAllProjects>
```

Comment:
This is adding a path to a Directory.Build.props in the parent folder.  It should probably be:

```xml
<MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileDirectory)Directory.Build.props</MSBuildAllProjects>
```